js/adapt-contrib-flipcard.js

```diff
@@ -166,42 +138,15 @@ define([
         // This function will be responsible to perform Single flip on flipcard where
         // only one card can flip and stay in the flipped state.
         performSingleFlip: function($selectedElement) {
-            var $flipcardContainer = $selectedElement.closest('.flipcard-widget');
-            if (!Modernizr.testProp('transformStyle', 'preserve-3d')) {
-                var $flipcardItem = $selectedElement.closest('.flipcard-item');
-                var $frontflipcard = $selectedElement.find('.flipcard-item-front');
-                var $backflipcard = $selectedElement.find('.flipcard-item-back');
-                var flipTime = this.model.get('_flipTime') || 'fast';
-
-                if ($backflipcard.is(':visible')) {     
-                    $backflipcard.fadeOut(flipTime, function() {
-                        $frontflipcard.fadeIn(flipTime);
-                        $flipcardItem.removeClass('flipcard-flip');
-                    });
-                } else {
-                    var $visibleflipcardBack = $flipcardContainer.find('.flipcard-item-back:visible');
-                    if ($visibleflipcardBack.length > 0) {
-                        $visibleflipcardBack.fadeOut(flipTime, function() {
-                            $flipcardContainer.find('.flipcard-item-front:hidden').fadeIn(flipTime);
-                            $flipcardItem.removeClass('flipcard-flip');
-                        });
-                    }
-                    $frontflipcard.fadeOut(flipTime, function() {
-                        $backflipcard.fadeIn(flipTime);
-                        $flipcardItem.addClass('flipcard-flip');
-                    });
-                }
-            } else {
-                if ($selectedElement.hasClass('flipcard-flip')) {
-                    $selectedElement.removeClass('flipcard-flip');
-                } else {
-                    $flipcardContainer.find('.flipcard-item').removeClass('flipcard-flip');
-                    $selectedElement.addClass('flipcard-flip');
-                }
-            }
+            var $items = this.$('.flipcard-item');
+            var shouldFlip = !$selectedElement.hasClass('flipcard-flip');
 
-            var flipcardElementIndex = this.$('.flipcard-item').index($selectedElement);
-            this.setVisited(flipcardElementIndex);
+            $items.removeClass('flipcard-flip');
```

Comment:
Should this remain conditional? ie
`shouldFlip && $items.removeClass('flipcard-flip');`

and then below be a toggle regardless, rather than a conditional addClass?

Reply:
When `.performSingleFlip` is called all cards should be unflipped regardless hence:

`$items.removeClass('flipcard-flip');`

Then, if $selectedElement should have the class added, if it was initially unflipped

Reply:
When `.performSingleFlip` is called all cards should be unflipped regardless hence:

`$items.removeClass('flipcard-flip');`

Then, if $selectedElement should have the class added, if it was initially unflipped